lib/saveable_extraction.rb

```diff
@@ -85,7 +85,7 @@ def process_multiple_choice(choice_ids, rationales = {}, is_final = nil)
         logger.error("Missing Embeddable::MultipleChoiceChoice id: #{choice_ids.join(",")}")
       end
       if ! multiple_choice
-        logger.error("Missing Embeddable::MultipleChoice id: #{choice.multiple_choice_id}")
```

Comment:
In order to capture more information here it would be better to do:
```
if ! choice
  ...
elsif ! multiple_choice
  logger.error("Missing Embeddable::MultipleChoice id: #{choice.multiple_choice_id}")
end
```
It is possible that on line 68 a choice is found, but then on line 69 the choice.multiple_choice is nil. 

I'd assume that is what the author was trying to capture here.